src/main/java/wolox/training/models/User.java

```diff
@@ -0,0 +1,99 @@
+package wolox.training.models;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToMany;
+import javax.validation.constraints.NotNull;
+import wolox.training.exceptions.BookAlreadyOwnedException;
+
+/**
+ * Model to the table Users.
+ */
+@Entity(name = "users")
+public class User {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE)
+    private long id;
+
+    @NotNull
+    private String userName;
+
+    @NotNull
+    private String name;
+
+    @NotNull
+    private LocalDate birthdate;
+
+    @NotNull
+    @ManyToMany
```

Comment:
We are not specifying a [cascade type](https://www.baeldung.com/jpa-cascade-types) here, and by default it will use no cascade type. There are several cascade types: Persist, Merge, Refresh, Delete and Detach (And there is `ALL`, which contains all of the mentioned ones). We should choose the ones that work better for our modeling :) 

Detach is only so that something gets detached along the cascade when you detach an object from the persistence context manually. This almost never happens, so we won't focus on this point.

Persist says: If you persist the Users instance to the DB, then persist every item that this collection has in the DB as well.
Delete says: If you delete the Users instance from the DB, then delete every item that this collection has in the DB as well.
Merge says: If you changed the in-memory values of the collection, then merge this changes to the DB so these are reflected there.
Refresh says: If something changed in the database values of the collection, then refresh those changes in the in-memory values.